src/audio.h

```diff
@@ -73,6 +73,7 @@ class NxAudioEngine
 	float getMusicVolume();
 	void setMusicVolume(float _volume, size_t time = 0);
 	void setMusicSpeed(float speed);
+	void setMusicLooping(bool looping);
```

Comment:
Maybe we can set this parameter to `false` by default, in order to match your previous line :)